### DISCUSSION TITLE
How can I disable CI failure when there is only a warning severity?

### DISCUSSION THREAD

**Discussion by trungdlp-wolffun** [golangci/golangci-lint#3829] [Q&A]
Same above

Results:
- Warning ....
- Warning ....
- Warning ....
- Warning ....
Error: issues found
  Ran golangci-lint in 3311ms

**Comment by bombsimon** [comment]
It depends on what the warnings are but the simple answer is that you probably don't. This is a result of #2357 and the fix in #2471. 

If you don't care about the exit code you'll have to analyze STDOUT manually. Either just `golangci-lint run || true` or some more sophisticated way that checks for warnings but not errors.
